spec/support/ansible_shared/automation_manager/refresher.rb

```diff
@@ -121,8 +126,7 @@ def assert_credentials
       :name   => "hello_machine_cred",
       :userid => "admin",
     )
-    expect(machine_credential.options.keys).to match_array([:become_method, :become_password, :become_username, :ssh_key_data, :ssh_key_unlock, :vault_password])
-    expect(machine_credential.options.keys).to match_array(machine_credential.class::EXTRA_ATTRIBUTES.keys)
```

Comment:
So this isn't assertable anymore for `machine_credential`?

Reply:
It is still assertable. L125 was an unintended duplicate and so removed this time. L129 is the modified L124